src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs

```diff
@@ -136,7 +135,7 @@ protected override void OnNavigatedTo(NavigationEventArgs eventArgs)
 
 			var parameters = (NavigationArguments)eventArgs.Parameter;
 			if (parameters.IsLayoutSwitch)
-				ReloadItemIconsAsync();
+				await ReloadItemIconsAsync();
```

Comment:
Shouldn't we leave it as fire and forget?

Reply:
Is there any downside to using `await`? If we remove it, should we suppress the warning and add a comment?

Reply:
Await will cause the process to pause there, and no further processing will take place until icons are reloaded. If it is working fine now, this modification may lead to performance degradation.
If you just want to get rid of the warning, you can write the following:
```csharp
_ = ReloadItemIconsAsync();
```

Reply:
👍